scripts/generate_contributors.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+/*
+ * This script fetches a maximum of PAGE_COUNT * 30 contributors and writes
+ * their usernames and url to the Contributors.md file.
+ */
+
+const fetch = require('node-fetch');
+const fs = require('fs');
+
+const PAGE_COUNT = 4;
```

Comment:
not exactly sure why the page count is set to 4? cant we get the number of `contributors / 30` to get this number?

Reply:
Each API call to `BASE_URL` retrieves 30 contributors. We currently have 97 distinct contributors. Having 4 makes sure we get all contributors.

I'll make this clearer in the code.

Reply:
Fixed! Let me know what you think.